pages/features.html

```diff
@@ -6,92 +6,122 @@
 
 {% block body %}
 <div class="content">
-  <H1>{% trans "World-Class Mixing Engine" %}</H1>
-  {% trans "Don't just beatmix, beatmix better." %}
+  <h1>{% trans "Precise Mixing Engine" %}</h1>
+  {% trans "Essential tools for seemlessly blending music" %}
 
   <div style="clear: both;"></div>
   <div class="halfbox_left">
-    <H2>{% trans "Four Decks, Advanced Controls" %}</H2>
+    <h2>{% trans "Four Decks" %}</h2>
     <p>
-      {% trans "Drop a song onto a deck in Mixxx to kick off your next mix. Each deck features a scratchable, scrolling waveform that marks beats and cue points, along with a whole-song waveform overview for quick seeking." %}
+      {% trans "Drop a song onto a deck in Mixxx to start playing right away. Each deck features a scratchable, scrolling waveform that marks beats and cue points, along with a whole-song waveform overview for quick seeking." %}
```

Comment:
This paragraph changes from a command style to description style. 
Maybe we can continue like "Scratch the track using mouse ...